english/voldemort/voldemort.md

```diff
@@ -0,0 +1 @@
+Harry and Hermoine resisted the temptation to go to go to Hogsmeade or Hagrid's. Instead they focused on tracking down Voldemort. They knew that he could only be in a few places but where would they start? 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.